apps/council-frontend/src/ui/delegate/DelegatesList/DelegatesList.tsx

```diff
@@ -80,6 +99,11 @@ function DelegatesList({
         </div>
       </div>
 
+      <DelegateSearchBar
+        onDelegateSearchInputUpdate={updateInputRef}
+        onDelegateListFiltering={handleDelegateListFiltering}
+      />
+
```

Comment:
Using a ref here is kind of confusing to me. If we need to raise the input filter string to the `DelegateList` state we can just have an `onChange(newFilter: string)` like function that sets to a `useState` hook in the `DelegateList` component.